### PR TITLE
Header issues

### PR DESCRIPTION
**Pull request by padmashreeshetty123**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://header-issues--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -838,19 +848,20 @@ header nav .nav-tools {
   }
 
   header .nav-menu-overlay .nav-menu-column.right > .submenu > li {
-   width: calc(50% - 20px);
+    width: calc(45vw - 4vw);
```
Comment (thomsebastin) [inline blocks/header/header.css:851]: Can be 41vw right?

Comment (padmashreeshetty123) [reply blocks/header/header.css:851]: yes. I was trying with different metrics. I will update

Comment (padmashreeshetty123) [reply blocks/header/header.css]: forgot to remove calc

File: package.json [hunk 2]
```diff
@@ -21,12 +21,12 @@
...
     "@babel/eslint-parser": "7.24.1",
-    "eslint": "8.57.0",
+    "eslint": "^8.57.0",
     "eslint-config-airbnb-base": "15.0.0",
     "eslint-plugin-import": "2.29.1",
     "eslint-plugin-json": "3.1.0",
     "eslint-plugin-xwalk": "github:adobe-rnd/eslint-plugin-xwalk#v0.1.0",
-    "stylelint": "16.6.1",
+    "stylelint": "^16.6.1",
     "stylelint-config-standard": "36.0.0"
```
Comment (thomsebastin) [inline package.json:30]: as discussed, please revert this.

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**